dotnetapp/dotnetapp.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
Note this is for the dockerhub branch.  This is the sample we build an image for that is pushed to microsoft/dotnet.  The samples we expect users to widely view are in the master branch.  That being said, the necessary VS support only exists in VS 2017 AFAIK.